hedera-node/src/main/java/com/hedera/services/context/SideEffectsTracker.java

```diff
@@ -371,6 +388,14 @@ public long getNetHbarChange() {
 		return netHbarChange;
 	}
 
+	public CurrencyAdjustments getStakingRewardsPaid() {
+		numRewardedAccounts = purgeZeroChanges(rewardedAccounts, rewardAmounts, numRewardedAccounts);
```

Comment:
There's no reason to call `purgeZeroChanges` because rewards are never negative.

Reply:
Agreed on purgeZeroChanges, but we would have -ve amount for account 800 in this list right ?

Reply:
I was thinking this would just be the +ve amounts paid (since they must come from `0.0.800`, that debit is implicit).